src/Runtime/ExecutionSession.cpp

```diff
@@ -63,9 +63,8 @@ ExecutionSession::ExecutionSession(
   }
 }
 
-std::vector<std::unique_ptr<OMTensor, decltype(&omTensorDestroy)>>
-ExecutionSession::run(
-    std::vector<std::unique_ptr<OMTensor, decltype(&omTensorDestroy)>> ins) {
+std::vector<OMTensorUniquePtr> ExecutionSession::run(
+    std::vector<OMTensorUniquePtr> ins) {
```

Comment:
I like the name `OMTensorUniquePtr`, will use it as well.

Reply:
I changed it in your recent code as well.